Metalama.Framework.Engine/Transformations/ReplaceWithFieldTransformation.cs

```diff
@@ -8,18 +8,19 @@
 
 namespace Metalama.Framework.Engine.Transformations;
 
-internal sealed class PromoteFieldTransformation : IntroducePropertyTransformation, IReplaceMemberTransformation
+internal sealed class ReplaceWithFieldTransformation : IntroduceFieldTransformation, IReplaceMemberTransformation
```

Comment:
Note that the diff is misleading here: `PromoteFieldTransformation` was renamed (and generalized) to `ReplaceWithPropertyTransformation`, while `ReplaceWithFieldTransformation` is a new type.